API.md

```diff
@@ -56,11 +56,11 @@ This page documents the Party service API endpoints. All endpoints return an `HT
         "cell_no": 1,     
         "formtype": "0001",
         "currency": "H",
-        "name": "Bolts and Ratchets Ltd"
```

Comment:
Does this make it a breaking change by removing the name from the attribute?

Reply:
In the short term that attribute will be duplicated in both places, but eventually 'name' will be removed from 'attributes' as part of the bigger party service refactoring. This has been raised to teams that use this endpoint.